zebra-rpc/src/server/error.rs

```diff
@@ -0,0 +1,118 @@
+//! RPC error codes & their handling.
+
+/// Bitcoin RPC error codes
+///
+/// Drawn from <https://github.com/zcash/zcash/blob/master/src/rpc/protocol.h#L32-L80>.
+///
+/// ## Notes
+///
+/// - All explicit discriminants fit within `i64`.
+#[derive(Default)]
+pub enum LegacyCode {
+    // General application defined errors
+    /// `std::exception` thrown in command handling
+    #[default]
+    Misc = -1,
+    /// Server is in safe mode, and command is not allowed in safe mode
+    ForbiddenBySafeMode = -2,
```

Comment:
We could just remove the ones we are not using and suggest to be added on demand. However as i am not sure what is better i am ok keeping it like it is.

Reply:
Not a biggie, so I'll leave it as it is.